src/drivers/PortFolder/test/PendingStatusError.sarif

```diff
@@ -0,0 +1,21 @@
+{

```

Comment:
If this is a template file, please name it as such.

Reply:
It is not a template file. I sent email yesterday explaining what it is. Basically, if expected and analysis results match you will see this. Otherwise, the mismatch will be shown. Example is attached in the coordination email thread.